CDCgov/prime-reportstream
# Updated so that bad phone numbers no longer stop the message from bei…

…ng sent.

This PR allows messages to be sent even if the phone number is bad. 

Test Steps:
1. Update the phone number in prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_truncate_CA_20230406-0001.fhir to 1 (000)000-0000
2. Run ./prime fhirdata --input-file src/testIntegration/resources/datatests/FHIR_to_HL7/sample_truncate_CA_20230406-0001.fhir -s metadata/hl7_mapping/receivers/STLTs/CA/CA.yml --output-format HL7 --output-file src/testIntegration/resources/datatests/FHIR_to_HL7/testhl7.hl7
3. Make sure it runs without error

## Changes
- try/catch added to allow messages to be sent even if the phone number is bad. 
- tests updated

## Checklist

### Testing

## Linked Issues
- #9644
